src/test/java/co/cask/wrangler/steps/PipelineTest.java

```diff
@@ -93,4 +95,26 @@ public void testSplitWithNull() throws Exception {
     Assert.assertEquals("1,2,a,A", row.getValue("firstCol"));
     Assert.assertNull(row.getValue("secondCol"));
   }
+
+  @Test
+  public void testEscapedStrings() throws Exception {
+    List<Step> steps = new ArrayList<>();
+    Row row = new Row("__col", StringEscapeUtils.unescapeJava("1\\ta"));
+
+    TextSpecification ts = new TextSpecification("set format csv \\t false\n" +
```

Comment:
It has to do with how the parsing happens when the specifications come from the UI. The UI translates the "tab" character into "\t" but keeps the "newline" as a newline character.